src/_layouts/default.html

```diff
@@ -21,8 +21,32 @@
       <script src="/assets/js/redirect.js"></script>
     {% endif %}
 
-    <link rel="stylesheet" href="/assets/main.css" />
-    <script defer data-domain="playbook.dxw.com" src="https://plausible.io/js/plausible.js"></script>
+    <link rel="stylesheet" href="/assets/main.css"/>
+    <script defer data-domain="playbook.dxw.com" src="https://plausible.io/js/script.manual.js"></script>
+    <!-- define the `plausible` function to manually trigger events - as the pageview event is not being sent automatically with 'script.manual.js' -->
+    <script>
+      window.plausible = window.plausible || function () {
+        (window.plausible.q = window.plausible.q || []).push(arguments)
+      }
+    </script>
+
+    <!-- trigger pageview -->
+    <script>
+      function prepareUrl(params) {
+        const url = new URL(location.href)
+        const queryParams = new URLSearchParams(location.search)
+        let customUrl = url.protocol + "//" + url.hostname + url
+          .pathname
+          .replace(/\/$/, '')
+        for (const paramName of params) {
+          const paramValue = queryParams.get(paramName)
+          if (paramValue) 
+            customUrl = customUrl + '/' + paramValue
```

Comment:
I think you fixed this in the next commit, but it might be good for it to be fixed here to keep the commit history green

```javascript
          if (paramValue) {
            customUrl = customUrl + '/' + paramValue
          }
```